dll/win32/winmm/CMakeLists.txt

```diff
@@ -1,6 +1,5 @@
 
 add_definitions(-D_WINE)
-include_directories(BEFORE ${REACTOS_SOURCE_DIR}/sdk/include/reactos/wine)
```

Comment:
Here and in a few other modules, you removed `include_directories()`, without calling the new function.
This seems to be causing
`/home/runner/work/reactos/reactos/src/sdk/include/psdk/winternl.h:31:2: error: #error "Do not use this header, use NDK!"`
and likely a few other errors.